.storybook/main.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   stories: ["../src/**/*.stories.tsx"],
-  addons: ["@storybook/addon-links", "@storybook/addon-essentials", "storybook-addon-performance/register"],
```

Comment:
This addon, for some reason, started cause TypeErrors / `state is not defined` errors when trying to load any story, so I had to remove it... :shrug: ?

Reply:
Yep, done